docs/general/administration/hardware-acceleration/intel-hwa-tutorial.md

```diff
@@ -0,0 +1,959 @@
+   ```
+
+3. Enable QSV or VA-API in Jellyfin and uncheck the unsupported codecs.
+
+
+
+#### LXC And LXD Container
+
+:::caution
+
+This has been tested with LXC 3.0 and may or may not work with older versions.
+
+:::
+
+1. Query the `render` and `video` group id on the host system.
+
+   :::note
+
+   On some releases, the group may be `input` instead of `render` and `video`.
+
+   :::
+
+   ```shell
+   $ getent group render | cut -d: -f3
+   $ getent group video | cut -d: -f3
+   ```
+
+2. Install the required drivers on the host system.
+
+3. Add your GPU to the container:
+
+   ```shell
+   $ lxc config device add <CONTAINER_NAME> gpu gpu gid=<GID_OF_HOST_RENDER_OR_VIDEO_GROUP>
+   ```
+
+4. Make sure you have the requied devices within the container:
+
+   ```shell
+   $ lxc exec jellyfin -- ls -l /dev/dri
+
+   total 0
+   crw-rw---- 1 root video 226,   0 Jun  4 02:13 card0
+   crw-rw---- 1 root video 226,   0 Jun  4 02:13 controlD64
+   crw-rw---- 1 root video 226, 128 Jun  4 02:13 renderD128
+   ```
+
+5. Configure Jellyfin to use QSV or VA-API acceleration and change the default GPU renderD128 if necessary.
+
+
+
+#### LXC On Proxmox
+
+:::note
+
+Jellyfin needs to run in a **privileged** LXC container.
+
+An existing unprivileged container can be converted to a priviledged container by taking a backup and restoring it as priviledged.
+
+:::
+
+1. Install the required drivers on the Proxmox host.
+
+2. Add your GPU to the container by editing `/etc/pve/lxc/<CONTAINER_ID>.conf`.
+
+   You may need to change the GIDs in the examples below to match those used on your host.
+
+   :::caution
+
+   This has been tested on Proxmox VE 7.1 - on previous versions you may need to change `cgroup2` to `cgroup`.
+
+   :::
+
+   ```conf
+   lxc.cgroup2.devices.allow: c 226:0 rwm
+   lxc.cgroup2.devices.allow: c 226:128 rwm
+   lxc.mount.entry: /dev/dri/card0 dev/dri/card0 none bind,optional,create=file
+   lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file
+   ```
+
+3. Restart your container and install the required drivers in your container.
+
+4. Add `jellyfin` user to the `video`, `render` and/or `input` groups depending on who owns the device inside the container.
+
+5. Configure Jellyfin to use QSV or VA-API acceleration and change the default GPU `renderD128` if necessary.
+
+
+
+### Verify On Linux
+
+:::note
+
+Root permission is required.
+
+:::
+
+1. Install the `intel-gpu-tools` package, which is used for debugging Intel graphics driver on Linux. The name varies with different distros.
+
+   - On Debian & Ubuntu:
+
+     ```shell
+     # apt update && apt install -y intel-gpu-tools
+     ```
+
+   - On Arch Linux:
+
+     ```shell
+     # pacman -Sy intel-gpu-tools
+     ```
+
+2. Play a video in Jellyfin web client and trigger a video transcoding by setting a lower resolution or bitrate.
+
+3. Use `intel_gpu_tool` command to check the occupancy of the engines as follows:
```

Comment:
```suggestion
3. Use `intel_gpu_top` command to check the occupancy of the engines as follows:
```

I think that's this command?

Reply:
Correct it's my bad. Will fix in the next push.